cuegui/cuegui/plugins/AttributesPlugin.py

```diff
@@ -203,6 +203,14 @@ def addData(parent, value):
         self.addTopLevelItem(root)
         self.expandAll()
 
+        self.itemSelectionChanged.connect(self.itemSingleClickedCopy)
+
+    def itemSingleClickedCopy(self):
+        selected = self.selectedItems()
+
+        if selected:
+            QtWidgets.QApplication.clipboard().setText(str(selected[0].text(1)))
```

Comment:
My only question was for this line -- since we're only using the first item selected, I was wondering if we should change this to use ALL selected items and join them, perhaps using newlines.

However, unless I'm missing something, it looks like it's not possible do select multiple items. So that suggestion is moot for now and not really testable anyway.